sound/soc/sof/sof-priv.h

```diff
@@ -472,6 +475,8 @@ int snd_sof_runtime_idle(struct device *dev);
 int snd_sof_resume(struct device *dev);
 int snd_sof_suspend(struct device *dev);
 void snd_sof_d0i3_entry_work(struct work_struct *work);
+void snd_sof_set_dsp_d0i0(struct snd_sof_dev *sdev, bool force, bool reload);
+void snd_sof_set_dsp_d0i3(struct snd_sof_dev *sdev);
```

Comment:
set state, don't duplicate the work with multiple functions

Reply:
It is quite different for setting d0i0(no delayed work needed) and d0i3 in current implementation, let me figure out the runtime-PM solution first.